build.sbt

```diff
@@ -65,6 +65,8 @@ mappings in Universal ++= {
   }
 }.value
 
+Universal / javaOptions ++= Seq("-XX:+UseG1GC", "-XX:+UseStringDeduplication", "-Xmx2G")
```

Comment:
double check what's the memory in both docker and kubernetes. The -Xmx2G cannot be higher than the memory limit that is being set

Reply:
you shouldn't need `Xmx` if the other parameters are working properly.